modules/navier_stokes/include/base/NSFVBase.h

```diff
@@ -557,8 +557,9 @@ NSFVBase<BaseType>::commonMomentumEquationParams()
       "'porous_medium_treatment' is enabled, the coefficients already contain a velocity "
       "multiplier but they are not multiplied with density yet!");
 
-  params.addParam<bool>(
-      "standard_friction_formulation", true, "Flag to enable standard friction formulation");
+  params.addParam<bool>("standard_friction_formulation",
+                        true,
+                        "Flag to enable the standard friction formulation or it's alternative.");
```

Comment:
```suggestion
                        "Flag to enable the standard friction formulation or its alternative.");
```
describe what the alternative here, a simplified model (see user documentation)